### PR TITLE
Update troubleshooting.asciidoc

### PR DESCRIPTION
**Pull request by mostlyjason**
Updating path to the logs. @ph can you confirm this is correct? This is what I saw in my linux tar.gz install.

### CODE REVIEW

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 1]
```diff
@@ -211,10 +211,10 @@ it might not be started. You need to <<run-elastic-agent,start {agent}>>.
 [[where-are-the-agent-logs]]
 == Where does {agent} store logs after startup?
 
-When started successfully, {metricbeat} logs are stored in
-`data/logs/metricbeat` under the folder where {agent} was started. If that log
-path does not exist, {agent} was unable to start {metricbeat}, which is a
-higher level problem to triage.
+When started successfully, {agent} logs are stored in the the folder where {agent}
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:214]:
```suggestion
When started successfully, {agent} logs are stored in the folder where {agent} 
```

**Comment by dedemorton** [comment]
@ph Should I merge this when the CI passes (even though I don't think it's accurate for all platforms (as noted in my previous comment). Not sure if Jason is planning to add info about the other platforms.

**Comment by ph** [comment]
@dedemorton good catch, theses path are only valid for the tarball and not the installed agent on windows, linux.

**Comment by dedemorton** [comment]
Closing this issue because we need to document the log locations for all platforms, not just macOS.

Here's the related issue: elastic/observability-docs#119

**Pull request closed without merging**